frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -798,7 +798,9 @@
 
         edit-color
         (fn [new-color]
-          (let [updated-color (merge new-color (select-keys color [:id :file-id :name]))]
+          (let [old-data (-> (select-keys color [:id :file-id])
+                             (assoc :name (str/join "/" [(:path color "") (:name color)])))
```

Comment:
There are helper functions to manage paths. In this case this could be
`(cp/merge-path-item (:path color) (:name color))`

Reply:
OK, thanks